client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -2135,14 +2135,13 @@ export default {
     },
 
     showInlineActions (field) {
-      const canInlineEdit = this.options.inlineEditFields.some(f => f.fieldID === field.moduleField.fieldID)
-      const canShowActions = this.showInlineEdit(field) || this.showInlineFilter(field)
-
-      return canInlineEdit && canShowActions
+      return this.showInlineEdit(field) || this.showInlineFilter(field)
     },
 
     showInlineEdit (field) {
-      return this.options.inlineRecordEditEnabled && field.canEdit && !this.showingDeletedRecords
+      const isFieldInlineEditable = this.options.inlineEditFields.some(fieldID => fieldID === field.moduleField.fieldID || fieldID === field.moduleField.name)
```

Comment:
Please make this a function and call it at the end of the 2144 line (&& isFieldInlineEditable)
Also make sure if no fields are selected, that means all of them have it enabled.